dist-git-to-src-git/README.md

```diff
@@ -68,13 +68,13 @@ Or breaking it down:
 
     $ cd git.centos.org
     $ dist2src checkout rpms/rpm c8s
-    $ dist2src get-archive rpms/rpm
```

Comment:
Although `extract-archive` will make sure the archive was downloaded, maybe it would make sense to keep it here, just to be explicit.